x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/esnative/tool/CommandLineHttpClient.java

```diff
@@ -131,7 +131,7 @@ public HttpResponse execute(String method, URL url, String user, SecureString pa
         return responseBuilder.build();
```

Comment:
Nit: Can we also please change the package name of `CommandLineHttpClient` out from `authc/esnative/tool` to reflect the new usage? I suggest `org.elasticsearch.xpack.security.tool` .